openedx_proversity_reports/api/v0/views.py

```diff
@@ -17,12 +17,19 @@
     generate_completion_report,
     generate_last_page_accessed_report,
     generate_time_spent_report,
+    generate_learning_tracker_report,
 )
 
 logger = logging.getLogger(__name__)
+TASKS = {
```

Comment:
I think this should come from the plugin settings. @andrey-canon

Reply:
done